client/my-sites/plan-features/header.jsx

```diff
@@ -229,18 +229,20 @@ export class PlanFeaturesHeader extends Component {
 			translate,
 			planType,
 			currentSitePlan,
-			annualPricePerMonth,
 			isInSignup,
 			isMonthlyPlan,
+			relatedYearlyPlan,
+			isLoggedInMonthlyPricing,
 		} = this.props;
 
-		if ( isInSignup && isMonthlyPlan && annualPricePerMonth < rawPrice ) {
+		if ( ( isInSignup || isLoggedInMonthlyPricing ) && isMonthlyPlan && relatedYearlyPlan ) {
+			const annualPricePerMonth = relatedYearlyPlan.raw_price / 12;
 			const discountRate = Math.round( ( 100 * ( rawPrice - annualPricePerMonth ) ) / rawPrice );
 			return translate( `Save %(discountRate)s%% by paying annually`, { args: { discountRate } } );
 		}
 
-		if ( isInSignup && ! isMonthlyPlan ) {
-			return translate( 'billed annually' );
+		if ( ( isInSignup || isLoggedInMonthlyPricing ) && ! isMonthlyPlan ) {
+			return translate( 'Billed annually' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'per month, billed annually' )` *ES Score: 10*